policyengine_us/variables/gov/states/ut/tax/income/taxable_income/ut_taxable_income.py

```diff
@@ -8,6 +8,7 @@ class ut_taxable_income(Variable):
     unit = USD
     documentation = "Form TC-40, line 9"
     definition_period = YEAR
-    adds = ["ut_total_income"]
     defined_for = StateCode.UT
-    subtracts = ["ut_subtractions_from_income", "ut_state_tax_refund"]
+
+    adds = ["ut_total_income"]
+    subtracts = ["ut_subtractions", "ut_state_tax_refund"]
```

Comment:
Should this be capped at 0? I can imagine that even total income - subtractions should be

Reply:
I add the capped 0 condition in `ut_income_tax_before_credits`, as it is represented in the tax form.